src/main/java/com/tournament/cup/play/TeamsListEight.java

```diff
@@ -0,0 +1,53 @@
+package com.tournament.cup.play;
+
+import com.tournament.cup.details.Squad;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TeamsListEight {
+    List<Squad> squads = new ArrayList<>();
+
+    Squad MU = new Squad("Manchester United", "Ronaldo", "Rangnick");
```

Comment:
as general java convention, variable names should come in camelCase (https://en.wikipedia.org/wiki/Camel_case). I this case since this never changes I suggest you to declare it like constant:
```
private static final Squad MU = new Squad("Manchester United", "Ronaldo", "Rangnick");
```